internal/pkg/term/color/color.go

```diff
@@ -13,16 +13,25 @@ import (
 	"github.com/fatih/color"
 )
 
-const colorEnvVar = "COLOR"
-
-var lookupEnv = os.LookupEnv
+// String markers to denote the status of an operation.
+var (
+	SuccessMarker = color.HiGreenString("✔")
+	ErrorMarker   = color.HiRedString("✘")
+)
```

Comment:
self note: I need to move these to a separate file so that they're rendered for windows